### DISCUSSION TITLE
Get All supported timezones

### DISCUSSION THREAD

**Discussion by rockstar84** [moment/luxon#1154] [General]
How to get all supported timezones with timezone name and offset

**Comment by icambron** [comment]
There's no way to do that in Luxon. The set of timezones that are supported are whatever the Intl API supports, and they don't provide a method to check that

**Reply by rockstar84** [reply]
My requirement is to populate dropdown with all timezones. momentjs provides moment.tz.names() to achieve same

**Reply by icambron** [reply]
You'll have to get the zone list from somewhere else; Luxon simply doesn't have access to such a list

**Comment by dasa** [comment]
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf

e.g., `Intl.supportedValuesOf('timeZone')`

**Reply by icambron** [reply]
Huh. Did this _just_ land? I actually had to restart Chrome to get the update that has this.

Well, OP, there's your answer!

**Reply by dasa** [reply]
I think Chrome 99 did get a lot of updates in Intl that aren't reflected in MDN yet: 
https://chromestatus.com/feature/5649454590853120
https://chromestatus.com/feature/5566859262820352

**Reply by thernstig** [reply]
How would one augment the list returned by `Intl.supportedValuesOf('timeZone')` with values such as `UTC` etc. that is supported on the current system? The docs around this are sparse. E.g. if someone wants to set `TZ` one would want the ones from `Intl.supportedValuesOf('timeZone')` as well as default ones like `UTC`.

**Reply by icambron** [reply]
All the things Luxon accepts are here: https://moment.github.io/luxon/#/zones?id=specifying-a-zone. If supportedValuesOf works correctly, only those values will be accepted under the "IANA" category of the table.

The "fixedOffset" category is trickier. Luxon accepts any fixed UTC offset, like, say, "UTC+3:35". So I don't think you can quite have an authoritative list. You'll have to use your judgment on which ones to include if you make a list.

**Comment by ab-pm** [comment]
Related: #353
